### PR TITLE
Remove the mesh linking for dae files

### PR DESCRIPTION
**Pull request by BenjaminHug8**
This PR is to revert the mesh linking for `.dae` files as they might have multiple material colors. This is currently not supported by the `Mesh` node of Webots.

### CODE REVIEW

File: urdf2webots/parserURDF.py [hunk 1]
```diff
@@ -28,9 +28,23 @@
 
 # to pass from external
 robotName = ''
-extensionList = ['.3ds', '.blend', '.bvh', '.dae', '.fbx', '.stl', '.obj', '.x3d']
+extensionListSingleColor = ['.stl', '.obj']
```
Comment (omichel) [inline urdf2webots/parserURDF.py:31]: I would rename this variable to make it clearer:
```suggestion
extensionListSingleAppearance = ['.stl', '.obj']
```

**Comment by BenjaminHug8** [comment]
Oh sorry I was sure that this PR was in `draft`.
I have to correct the tests.

**Comment by BenjaminHug8** [comment]
I also confirm that the most standard format for the mesh files used in URDF are `.stl`, `.dae` and `.obj`.

It has been rarely mentioned that `.fbx` and `.mesh` can also be used in URDF files, but they seem to be almost never used by the community.

**Comment by BenjaminHug8** [comment]
@omichel is it ok for you if after the merging of this PR I overwrite the version `1.0.11` as `.dae` conversion is not fully functional now?

**Review by omichel — APPROVED** [review]
I would like that @lukicdarkoo also review this PR before we merge it.

**Review by lukicdarkoo — COMMENTED** [review]
Looks good.

Could you add tests? It would be nice to have tests for .dae files, just to be able to quickly verify whether the .dae conversion works as expected.

**Comment by BenjaminHug8** [comment]
Addition of a test on a kuka arm with `.dae` and `.stl` mesh files.

Correction of conversion of `indexLineSet` to use `Appearance` instead of `PBRAppearance`.

**Review by lukicdarkoo — APPROVED** [review]
Perfect, thanks!

**Review by omichel — APPROVED** [review]

**Pull request merged**